web/Create a quiz using simple HTML and JS.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Quiz Website</title>
+</head>
+<body>
+    <form action="" id="MyForm">
+    <div>
+        <h1 id="Question">Sample Question? </h1>
+    </div>
+    <h2>
+    <div>
+        <input type="radio" id="option1" name="MCQ" value="0">
+        <label for="option1">Option1</label>
+    </div>
+    <div>
+        <input type="radio" id="option2" name="MCQ" value="1">
+        <label for="option2">Option2</label>
+    </div>
+    <div>
+        <input type="radio" id="option3" name="MCQ" value="2">
+        <label for="option3">Option3</label>
+    </div>
+    <div>
+        <input type="radio" id="option4" name="MCQ" value="3">
+        <label for="option4">Option4</label>
+    </div>
+    </div>
+        <input type="button" id="BUTTON">Submit</button>
```

Comment:
You can't close the input tag with a button tag, if u want to display text in a  input button, u can do this
```suggestion
        <input type="button" id="BUTTON" label="Submit">
```